src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/ActivityFunction/FunctionAnalyzer.cs

```diff
@@ -215,7 +214,7 @@ private ITypeSymbol GetActivityFunctionInputTypeName(SyntaxNodeAnalysisContext c
             if (SyntaxNodeUtils.TryGetParameterNodeNextToAttribute(context, attributeExpression, out SyntaxNode inputTypeNode))
             {
                 var inputType = context.SemanticModel.GetTypeInfo(inputTypeNode).Type;
-                if (inputType.ToString().Equals("Microsoft.Azure.WebJobs.IDurableActivityContext") || inputType.ToString().Equals("Microsoft.Azure.WebJobs.DurableActivityContext"))
+                if (inputType.ToString().Equals("Microsoft.Azure.WebJobs.Extensions.DurableTask.IDurableActivityContext") || inputType.ToString().Equals("Microsoft.Azure.WebJobs.DurableActivityContext"))
```

Comment:
Should we also check for [DurableActivityContextBase](https://github.com/Azure/azure-functions-durable-extension/blob/v1/src/WebJobs.Extensions.DurableTask/DurableActivityContextBase.cs)?